src/configure-apollo-network.js

```diff
@@ -1,90 +0,0 @@
-
-/**
- * Configure a given context to support Apollo
- *
- * If Apollo network settings are provided during render then we need to expose
- * some important objects to the vm context. ApolloNetwork should have two
- * properties: 'url' (for the URL of the GraphQL endpoint) and 'headers'
- * for a key-value map of headers to send to the GraphQL endpoint. This
- * can include things such as cookies and xsrf tokens.
- * Requests will automatically timeout after 1000ms, unless another
- * timeout is provided via a 'timeout' property.
- */
-const ApolloClient = require("apollo-client");
-const apolloCacheInmemory = require("apollo-cache-inmemory");
-const apolloLinkHttp = require("apollo-link-http");
-const fetch = require('node-fetch');
```

Comment:
It's not clear to me how we're going to handle this when we do SSR entirely from the client, since we need to access this library (but we definitely don't want to bundle it in our client build). Maybe we just need to provide a polyfill for it instead?

Reply:
I was thinking that we could have server-side code dynamically requested only when in the SSR context, and as such, it would never load for client side situations.

Reply:
But that's the next bit of my investigation 😄

Reply:
That said, long term, I think we can have a separate SSR bundle that would be the "bootstrap" script that kicks off the whole process. I think we have options.

Reply:
Are y'all talking about `node-fetch`?

Reply:
I am, yes.

Reply:
We have `fetch` on the client already.  Why would we need `node-fetch` specifically to do SSR on from client code?

Reply:
> We have `fetch` on the client already. Why would we need `node-fetch` specifically to do SSR on from client code?

We wouldn't. Hence why we want to make sure that when this code lives in Webapp, we don't bundle node-fetch with client code.

Reply:
(not working; I'm doing OSS so I was in GitHub...)